Foundation/src/DateTime.cpp

```diff
@@ -162,21 +145,13 @@ DateTime& DateTime::operator = (double julianDay)
 {
 	_utcTime = toUtcTime(julianDay);
 	computeGregorian(julianDay);
+	checkValid();
```

Comment:
IMO this one is redundant unless it checks the conversion.

Reply:
it makes sure the passed julian day gives a valid DateTime